docs/DeveloperGuide.md

```diff
@@ -283,28 +283,162 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `InterviewHub` and the **Actor** is the `hiring manager`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 Add an interview**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. Hiring manager requests to add an interview.
+2. InterviewHub adds the interview.
 
     Use case ends.
 
 **Extensions**
+* 1a. One of the user-provided parameters is invalid
+    * 1a1. InterviewHub shows an error message.
 
-* 2a. The list is empty.
+      Use case resumes at step 1.
 
-  Use case ends.
+**Use case: UC02 List all interview**
 
-* 3a. The given index is invalid.
+**MSS**
+
+1. Hiring manager requests to list all interviews.
+2. InterviewHub displays all scheduled interviews.
+
+    Use case ends.
+
+**Extensions**
+* 1a. The list is empty
+  * 1a1. InterviewHub shows an error message.
+  
+    Use case ends.
+
+**Use case: UC03 Delete an Interview**
+
+**MSS**
+
+1. Hiring manager <u> views the list of all interviews (UC02) </u>
```

Comment:
Perhaps an extension for this step would be when the list of interviews is empty? If that is the case I propose informing the user(in a nice way of course), that there are no interviews scheduled and therefore nothing to delete.

Reply:
Good point, added to the Week 8 disussion agenda